homeassistant/util/package.py

```diff
@@ -60,3 +60,27 @@ def check_package_exists(package: str, lib_dir: str) -> bool:
     # Check packages from global + virtual environment
     # pylint: disable=not-an-iterable
     return any(dist in req for dist in pkg_resources.working_set)
+
+
+def handle_requirements(component: Any, config_dir: str) -> bool:
```

Comment:
I don't think that this method is worth extracting. We should not have the logic that something has an attribute or key of `REQUIREMENTS`.

Let's just add the same logic to both places.